ADNKit/ANKClient+ANKStreamMarker.h not found when building as subproject

I'm trying to build head as a subproject of an OS X app Xcode project, and getting the following error:

```

In file included from /Users/tonyarnold/Documents/CocoaBots Projects/RandomADNApp/RandomADNApp/Classes/TCBCADNController.m:9:
In file included from /Users/tonyarnold/Documents/CocoaBots Projects/RandomADNApp/RandomADNApp/Classes/TCBCADNController.h:10:
In file included from /Users/tonyarnold/Library/Developer/Xcode/DerivedData/RandomADNApp-grjvyylajclibubwnwdibloxxqfe/Build/Products/Debug/ADNKit.framework/Headers/ADNKit.h:15:
In file included from /Users/tonyarnold/Library/Developer/Xcode/DerivedData/RandomADNApp-grjvyylajclibubwnwdibloxxqfe/Build/Products/Debug/ADNKit.framework/Headers/ANKClient.h:13:
In file included from /Users/tonyarnold/Library/Developer/Xcode/DerivedData/RandomADNApp-grjvyylajclibubwnwdibloxxqfe/Build/Products/Debug/ADNKit.framework/Headers/AFNetworking.h:35:
/Users/tonyarnold/Library/Developer/Xcode/DerivedData/RandomADNApp-grjvyylajclibubwnwdibloxxqfe/Build/Products/Debug/ADNKit.framework/Headers/AFHTTPClient.h:84:2: warning: SystemConfiguration framework not found in project, or not included in precompiled header. Network reachability functionality will not be available. [-W#warnings]
#warning SystemConfiguration framework not found in project, or not included in precompiled header. Network reachability functionality will not be available.
 ^
In file included from /Users/tonyarnold/Documents/CocoaBots Projects/RandomADNApp/RandomADNApp/Classes/TCBCADNController.m:9:
In file included from /Users/tonyarnold/Documents/CocoaBots Projects/RandomADNApp/RandomADNApp/Classes/TCBCADNController.h:10:
/Users/tonyarnold/Library/Developer/Xcode/DerivedData/RandomADNApp-grjvyylajclibubwnwdibloxxqfe/Build/Products/Debug/ADNKit.framework/Headers/ADNKit.h:28:9: fatal error: 'ADNKit/ANKClient+ANKStreamMarker.h' file not found
#import <ADNKit/ANKClient+ANKStreamMarker.h>
        ^
1 warning and 1 error generated.
```

--- a/ADNKit.xcodeproj/project.pbxproj
+++ b/ADNKit.xcodeproj/project.pbxproj
@@ -249,7 +249,7 @@
 		252AD7171745C76600E12492 /* ANKClient+ANKRequestsAPI.m in Sources */ = {isa = PBXBuildFile; fileRef = 252AD7131745C76600E12492 /* ANKClient+ANKRequestsAPI.m */; };
 		252AD7181745C76600E12492 /* ANKClient+ANKRequestsAPI.m in Sources */ = {isa = PBXBuildFile; fileRef = 252AD7131745C76600E12492 /* ANKClient+ANKRequestsAPI.m */; };
 		252AD7191745C76600E12492 /* ANKClient+ANKRequestsAPI.m in Sources */ = {isa = PBXBuildFile; fileRef = 252AD7131745C76600E12492 /* ANKClient+ANKRequestsAPI.m */; };
-		252E632517421632003143D6 /* ANKClient+ANKStreamMarker.h in Headers */ = {isa = PBXBuildFile; fileRef = 252E632317421632003143D6 /* ANKClient+ANKStreamMarker.h */; };
+		252E632517421632003143D6 /* ANKClient+ANKStreamMarker.h in Headers */ = {isa = PBXBuildFile; fileRef = 252E632317421632003143D6 /* ANKClient+ANKStreamMarker.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		252E632617421632003143D6 /* ANKClient+ANKStreamMarker.m in Sources */ = {isa = PBXBuildFile; fileRef = 252E632417421632003143D6 /* ANKClient+ANKStreamMarker.m */; };
 		252E632717421632003143D6 /* ANKClient+ANKStreamMarker.m in Sources */ = {isa = PBXBuildFile; fileRef = 252E632417421632003143D6 /* ANKClient+ANKStreamMarker.m */; };
 		252E632817421632003143D6 /* ANKClient+ANKStreamMarker.m in Sources */ = {isa = PBXBuildFile; fileRef = 252E632417421632003143D6 /* ANKClient+ANKStreamMarker.m */; };
@@ -354,7 +354,7 @@
 		5D2CFEE91703E8F200086FBD /* ANKTokenLimits.m in Sources */ = {isa = PBXBuildFile; fileRef = 5D2CFEE51703E8F200086FBD /* ANKTokenLimits.m */; };
 		5D955BE6174BD10B0077E387 /* ANKGeneralParameters.h in Headers */ = {isa = PBXBuildFile; fileRef = 5D955BE4174BD10B0077E387 /* ANKGeneralParameters.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		5D955BE7174BD10B0077E387 /* ANKGeneralParameters.h in Headers */ = {isa = PBXBuildFile; fileRef = 5D955BE4174BD10B0077E387 /* ANKGeneralParameters.h */; settings = {ATTRIBUTES = (Public, ); }; };
-		5D955BE8174BD10B0077E387 /* ANKGeneralParameters.h in Headers */ = {isa = PBXBuildFile; fileRef = 5D955BE4174BD10B0077E387 /* ANKGeneralParameters.h */; };
+		5D955BE8174BD10B0077E387 /* ANKGeneralParameters.h in Headers */ = {isa = PBXBuildFile; fileRef = 5D955BE4174BD10B0077E387 /* ANKGeneralParameters.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		5D955BE9174BD10B0077E387 /* ANKGeneralParameters.m in Sources */ = {isa = PBXBuildFile; fileRef = 5D955BE5174BD10B0077E387 /* ANKGeneralParameters.m */; };
 		5D955BEA174BD10B0077E387 /* ANKGeneralParameters.m in Sources */ = {isa = PBXBuildFile; fileRef = 5D955BE5174BD10B0077E387 /* ANKGeneralParameters.m */; };
 		5D955BEB174BD10B0077E387 /* ANKGeneralParameters.m in Sources */ = {isa = PBXBuildFile; fileRef = 5D955BE5174BD10B0077E387 /* ANKGeneralParameters.m */; };
@@ -1089,6 +1089,7 @@
 				252A891816F8CDF2001F3E8F /* ANKClient+ANKPost.h in Headers */,
 				252A891B16F8CDF2001F3E8F /* ANKClient+ANKPostStreams.h in Headers */,
 				252A891E16F8CDF2001F3E8F /* ANKClient+ANKStream.h in Headers */,
+				252E632517421632003143D6 /* ANKClient+ANKStreamMarker.h in Headers */,
 				252A892116F8CDF2001F3E8F /* ANKClient+ANKTokenStatus.h in Headers */,
 				252A892416F8CDF2001F3E8F /* ANKClient+ANKUser.h in Headers */,
 				252A892716F8CDF2001F3E8F /* ANKEntities.h in Headers */,
@@ -1097,6 +1098,7 @@
 				252A893016F8CDF2001F3E8F /* ANKFile.h in Headers */,
 				252A893316F8CDF2001F3E8F /* ANKFilter.h in Headers */,
 				252A893616F8CDF2001F3E8F /* ANKFilterClause.h in Headers */,
+				5D955BE8174BD10B0077E387 /* ANKGeneralParameters.h in Headers */,
 				252A893916F8CDF2001F3E8F /* ANKGeolocation.h in Headers */,
 				252A893C16F8CDF2001F3E8F /* ANKHashtagEntity.h in Headers */,
 				252A893F16F8CDF2001F3E8F /* ANKImage.h in Headers */,
@@ -1126,9 +1128,7 @@
 				5D2CFEE61703E8F200086FBD /* ANKTokenLimits.h in Headers */,
 				2567680F1711539D00C57D47 /* NSString+ANKAdditions.h in Headers */,
 				2567682717153E3200C57D47 /* ANKResourceMap.h in Headers */,
-				252E632517421632003143D6 /* ANKClient+ANKStreamMarker.h in Headers */,
 				252AD7161745C76600E12492 /* ANKClient+ANKRequestsAPI.h in Headers */,
-				5D955BE8174BD10B0077E387 /* ANKGeneralParameters.h in Headers */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};

